src/test/java/com/commercetools/emailprocessor/email/EmailProcessorTest.java

```diff
@@ -120,10 +134,17 @@ public void callApiEndpoint_validEndpointUrlIsGiven_shouldAddCorrectVariablesToR
         TenantConfiguration configuration = new TenantConfiguration();
         configuration.setHttpPost(httpPost);
         configuration.setProjectKey(tenantid);
+        configuration.setEncryptionKey(tenantConfiguration.getEncryptionKey());
```

Comment:
@ahalberkamp why do you not use the other constructor which takes these values as params in one line? instead of doing it in 4 lines with empty constructor and setters? Also, if some params, are optional, then better use the Builder pattern?

Reply:
line 131: `tenantid` -> `tenantId`

Reply:
line 133: final HttpPost,
line 134: final TenantConfiguration

Reply:
1b7fca7